inc/WooCommerce/WooCommerce.php

```
@@ -0,0 +1,194 @@
+	public function render_product_column( string $column, int $product_id ): void {
+		if ( 'try_aura_try_on' !== $column ) {
+			return;
+		}
+
+		$enabled = get_post_meta( $product_id, self::TRY_ON_META_KEY, true );
+		// Default to disabled if not set.
+		if ( empty( $enabled ) ) {
+			$enabled = 'no';
+		}
+
+		$checked = 'yes' === $enabled;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$checked'. (undefined)

(UnusedLocalVariable)